<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstWIndow</class>
 <widget class="QDialog" name="firstWIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">firstWindow{
	background-color: qlineargradient(spread:pad, x1:0.269528, y1:0.34, x2:1, y2:1, stop:0 rgba(0, 40, 143, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="3,10">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetForFutherUse" native="true"/>
       </item>
       <item>
        <widget class="QFrame" name="newworkFrame">
         <property name="styleSheet">
          <string notr="true">QFrame{
font: 700 12pt &quot;Microsoft YaHei UI&quot;;
color:rgb(230, 230, 255)}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="midLineWidth">
          <number>3</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>连接信息：</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout" columnstretch="1,4">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>IP地址：</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string> NULL</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>硬件ID：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>NULL</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="1">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>348</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="connectToHostButton">
                <property name="font">
                 <font>
                  <pointsize>15</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>连接</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>58</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="1">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <property name="midLineWidth">
          <number>4</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>冲击试验</string>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">QLabel{

font: 700 12pt &quot;Microsoft YaHei UI&quot;;
color:rgb(210, 206, 255)
}
</string>
         </property>
         <widget class="QWidget" name="page">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(81, 81, 81);</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>30</y>
             <width>431</width>
             <height>211</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>30</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="9,2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton_6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">
background-color: rgb(103, 207, 255);


</string>
                </property>
                <property name="text">
                 <string>icon</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>水平冲击碰撞试验</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5" stretch="9,2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton_7">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(59, 255, 49);</string>
                </property>
                <property name="text">
                 <string>icon</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>冲击响应谱实验</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
